common/page.html

```diff
@@ -6,13 +6,48 @@
 				<div id="legend">
 					<h2>Legend</h2>
 					<ul>
-						<li class="gpio"><span class="pin"></span> GPIO <small>(General Purpose IO)</small></li>
-						<li class="spi"><span class="pin"></span> SPI <small>(Serial Peripheral Interface)</small></li>
-						<li class="i2c"><span class="pin"></span> I<sup>2</sup>C <small>(Inter-integrated Circuit)</small></li>
-						<li class="uart"><span class="pin"></span> UART <small>(Universal Asyncronous Receiver/Transmitter)</small></li>
-						<li class="gnd"><span class="pin"></span> Ground</li>
-						<li class="pow5v"><span class="pin"></span> 5v <small>(Power)</small></li>
-						<li class="pow3v3"><span class="pin"></span> 3v <small>(Power)</small></li>
+						<li class="gpio">
+							<a href="/pinout/wiringpi" title="GPIO (General Purpose IO)">
```

Comment:
Why link to the WiringPi page?

Reply:
yeah, that's looks like arbitrary choice. I've picked this up not being able to decide where to point this link. There is no `gpio` topic page atm.